source/_integrations/cambridge_audio.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: Cambridge Audio
+description: Instructions on how to integrate Cambridge Audio Receivers into Home Assistant.
+ha_category:
+  - Media player
+ha_release: '2024.10'
+ha_iot_class: Local Push
+ha_domain: cambridge_audio
+ha_platforms:
+  - media_player
+ha_codeowners:
+  - '@noahhusby'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The **Cambridge Audio** {% term integration %} allows you to control all receivers and streamers that support the StreamMagic app.
+
+The integration automatically discovers all enabled zones and sources. Each zone is added as a media player device with the enabled sources available as inputs. Media information and transport controls are supported if the selected source reports it.
+
+## Supported devices
+
+This integration allows you to connect the following devices:
+
+- Cambridge Audio Evo 75
+- Cambridge Audio Evo 150
+- Cambridge Audio CXN
+- Cambridge Audio CXN (v2)
+- Cambridge Audio CXR120
+- Cambridge Audio CXR200
+- Cambridge Audio 851N
+- Cambridge Audio Edge NQ
+
+Older, RS-232 serial-based amplifiers like the [CXA series](https://www.cambridgeaudio.com/usa/en/products/hi-fi/cx-series-2/cxa81)
+use a different protocol and are not currently supported.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+    description: The IP address of your device can be found by navigating to the device on the [StreamMagic app](https://www.cambridgeaudio.com/usa/en/products/streammagic) and selecting `Settings` → `IP address`.
+    required: true
+    type: string
+{% endconfiguration_basic %}
+
+## FAQ
```

Comment:
```suggestion
## Troubleshooting
```
The topics listed below sound like troubleshooting information.

Reply:
Yep, they are. LGTM